modules/custom/activity_send/modules/activity_send_email/src/Plugin/EmailFrequency/Immediately.php

```diff
@@ -174,7 +174,7 @@ protected function sendEmail(string $body_text, string $langcode, User $target,
     ];
 
     // Construct the body & subject for email sending.
-    $params['body'] = $this->renderer->renderPlain($notification);
+    $params['body'] = $this->renderer->renderInIsolation($notification);
```

Comment:
Hi @vcsvinicius,
Please note that we just merged some fixes to this file, so you might need to update it. See here: https://github.com/goalgorilla/open_social/pull/4092/files